src/shipit_static_analysis/shipit_static_analysis/workflow.py

```diff
@@ -123,14 +124,22 @@ def run(self, revision):
         with stats.api.timer('runtime.mach'):
             # Only run mach if revision has any C/C++ files
             if revision.has_clang_files:
-                # mach configure with mozconfig
+                # Mach pre-setup with mozconfig
                 logger.info('Mach configure...')
                 run_check(['gecko-env', './mach', 'configure'], cwd=self.repo_dir)
```

Comment:
Unrelated to this PR: do you think we should measure the time for the mach steps to see where we can improve?

Reply:
All those steps are needed by `mach static-analysis`, i don't know what could improve this.
But adding more metrics is always a good idea i guess, so #1053

Reply:
Yeah we won't be able to do much, but we could file bugs on bugzilla to ask someone to improve those steps.